msg/BoundingBox2D.msg

```diff
@@ -1,9 +1,9 @@
 # Note: This message plans to move to geometry_msgs in the final implementation.
 
-# The x/y position and (optional) rotation of the bounding box center.
-geometry_msgs/Pose2D pose
+# The 2D position (in pixels) and orientation of the bounding box origin.
+geometry_msgs/Pose2D origin
 
-# (Optional) The size of the bounding box surrounding the object. The center of
-# the bounding box is the position of the pose.
-float32 bbox_size_x
-float32 bbox_size_y
\ No newline at end of file
+# The size (in pixels) of the bounding box surrounding the object relative 
+# to the pose of its origin
+uint32 size_x
+uint32 size_y
```

Comment:
Related to my previous comment: If we allow the bounding box origin to be given as floats with rotation, we should also allow floats for the size. The unit should still be pixels. However, I would prefer to remove the rotation and use uint32 everywhere (bounding box origin + size).